playground/bicep/BicepSample.AppHost/aspire-manifest.json

```diff
@@ -99,17 +108,16 @@
       "path": "aspire.hosting.azure.bicep.postgres.bicep",
       "params": {
         "serverName": "postgres2",
-        "administratorLogin": "someuser",
-        "administratorLoginPassword": "{password.value}",
+        "keyVaultName": "",
+        "administratorLogin": "{administratorLogin.value}",
+        "administratorLoginPassword": "{administratorLoginPassword.value}",
         "databases": [
           "db2"
-        ],
-        "keyVaultName": ""
+        ]
       }
     },
     "db2": {
-      "type": "azure.bicep.v0",
-      "connectionString": "{postgres2.connectionString};Database=db2",
+      "type": "postgres.database.v0",
```

Comment:
We want the connection string expressed in the manifest

Reply:
I haven't figured out how to do this yet without replacing the manifest publisher on the child resource. The intuitive thing to do is that in the manifest publishing callback for `postgres.database.v0` we just call `GetConnectionString()` ... we redirect to `AzurePostgresResource.GetConnectionString()` but that `GetConnectionString()` call doesn't know we are in manifest publishing mode and tries to return the actual connection string, not the manifest placeholder value.

Reply:
This is now `value.v0`.